features/step_definitions/form_steps.rb

```diff
@@ -29,7 +29,7 @@
 end
 
 And "field {string} {is} disabled" do |field, disabled|
-  assert_equal disabled, find_field(field).disabled?
+  assert_equal disabled, find_field(field, { disabled: :all }).disabled?
```

Comment:
`find_field` only finds fields that are **not** disabled by default, so this step never worked properly (but seems that it was not used anywhere :sweat_smile:)

Reply:
For a better error message, maybe something along the lines of

```suggestion
  disabled? ?  assert_selector(:field, field, { disabled: :all }) : assert_not_selector(:field, field, { disabled: :all }) 
```

Reply:
What would be the difference between the messages? :upside_down_face: 

I think the current message would still be useful enough, so I'd avoid having to duplicate that assertion.

Reply:
Presently it would be "expected true to be false" or something along these lines. With the assertion it will explain that element was or wasn't there.

Actually if field was not found, you will probably get "no method #disabled for nil class" or who knows what. Unless I don't understand your selector, which is possible, I didn't look at the html :)

While this may not be a big difference when issue is easily reproducible locally, the proper error message would save you a step or two of debugging what exactly we searched for and what we found.

Reply:
OK, you're right.

It was:
```
Minitest::Assertion: Expected: false
  Actual: true
- call stack here -
/home/dmayorov/Projects/3scale/porta/features/old/providers/settings.feature:30:in `field "Account approval required" is not disabled'
```

Now it is:

```
Capybara::ExpectationNotMet: expected to find field "Account approval required" that is not disabled but there were no matches
- call stack here -
/home/dmayorov/Projects/3scale/porta/features/old/providers/settings.feature:30:in `field "Account approval required" is not disabled'
```

or
```
Capybara::ExpectationNotMet: expected to find visible field "Account approval required" that is disabled but there were no matches. Also found "", which matched the selector but not all filters. Expected disabled true but it wasn't
- call stack here -
/home/dmayorov/Projects/3scale/porta/features/old/providers/settings.feature:24:in `field "Account approval required" is disabled'
```

There is also `assert_field` method, but it didn't work for me - looks like a bug, although not sure.

Reply:
Fixed in https://github.com/3scale/porta/pull/3666/commits/ff9bd97292c938d87c6993b3879b21600ff23227

Reply:
These `assert_field` and `assert_link`, etc. are broken. I think they come from minitest and probably we hit some incompatibility, didn't look much into that but I know what you mean.